### DISCUSSION TITLE
require full log for submission error "异常"

### DISCUSSION THREAD

**Discussion by Chaodong** [PaddlePaddle/PaddleSpatial#228] [BaiduKDDCup2022] [answered]
Team name is Dynamo and submit time is  2022-06-27 09:48

**Comment by ramber1836** [comment, marked as answer]
Some errors are as below: 

W0627 10:03:03.520191   844 system_allocator.cc:210] cudaHostAlloc failed.
W0627 10:03:03.520274   844 naive_best_fit_allocator.cc:573] cudaHostAlloc Cannot allocate 38707200 bytes in CUDAPinnedPlace


--------------------------------------
C++ Traceback (most recent call last):
--------------------------------------
0   std::thread::_State_impl<std::thread::_Invoker<std::tuple<ThreadPool::ThreadPool(unsigned long)::{lambda()#1}> > >::_M_run()
1   std::__future_base::_State_baseV2::_M_do_set(std::function<std::unique_ptr<std::__future_base::_Result_base, std::__future_base::_Result_base::_Deleter> ()>*, bool*)
2   void paddle::memory::Copy<phi::GPUPinnedPlace, phi::Place>(phi::GPUPinnedPlace, void*, phi::Place, void const*, unsigned long)
3   void paddle::memory::Copy<phi::Place, phi::Place>(phi::Place, void*, phi::Place, void const*, unsigned long, void*)
4   void paddle::memory::Copy<phi::GPUPinnedPlace, phi::CPUPlace>(phi::GPUPinnedPlace, void*, phi::CPUPlace, void const*, unsigned long)

----------------------
Error Message Summary:
----------------------
FatalError: `Segmentation fault` is detected by the operating system.
  [TimeInfo: *** Aborted at 1656295383 (unix time) try "date -d @1656295383" if you are using GNU date ***]
  [SignalInfo: *** SIGSEGV (@0x20) received by PID 502 (TID 0x7fc6a5089700) from PID 32 ***]

/home/aistudio/aistudio_eval_script: line 192:   502 Segmentation fault      (core dumped) ${python_bin} "${work_dir}/eval_main.py" --submit_file "${submit_file}"

**Reply by Chaodong** [reply]
Could you please tell me the online cuda version ?
I cannot reproduce the error offline. I searched the internet, it might be caused by cuda version.

**Reply by shaido987** [reply]
The verison used is 11.2, see: https://github.com/PaddlePaddle/PaddleSpatial/discussions/229
